<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku-Fun</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="inputFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>153</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="playerLabel">
       <property name="text">
        <string>Current Player:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rowLabel">
       <property name="text">
        <string>Row</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rowEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="colLabel">
       <property name="text">
        <string>Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="colEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="numLabel">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="submitBtn">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="gridFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="gridLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="playerInFrame">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>How much Player ?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="playerInEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="playerInBtn">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>371</width>
      <height>47</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="numToRemLabel">
       <property name="text">
        <string>Numbers to remove :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numToRemEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="genBtn">
       <property name="text">
        <string>Start Game</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>submitBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>genBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerInBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>player()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submit()</slot>
  <slot>generate()</slot>
  <slot>player()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
